app/code/Magento/ProductAlert/etc/adminhtml/di.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
```

Comment:
For layout updating (bocks) need to work with layout directives (with layout updates xml files)
But not with ObjectManager configuration

Reply:
ok. I'll find another decision.  Thanks.